### DISCUSSION TITLE
Toonsquid 2.0 is missing an import feature

### DISCUSSION THREAD

**Discussion by ArtsyStar1** [keiwando/toonsquid#712] [General]
As much as I’m excited for Toonsquid 2.0, I feel like there’s an important feature missing which is the support for photoshop(psd) files. 

I don’t use photoshop(at least not anymore) but some apps like ibispaint and procreate _do_ allow people to import and export layered psd files, it would be nice to see Toonsquid implementing this feature, especially with _vectorized_ files now that the svg import has been officially confirmed to be in the 2.0 update.

The only animation apps on iPad that tend to have support for psd files are Callipeg(which is more fit for hand drawn animations) and Animation Desk(same thing with Callipeg), I wonder if more features will be added in the future, however, since the 2.0 update is going to be very big, a psd file import(and export) would be extremely useful to many people, including myself.

**Comment by kavlor** [comment]
Yes this would be very useful especially with a choice to load into drawing or animation layers.
